marketplace_builder/relationship_transactable_types/special_group_transactables.yml

```diff
@@ -1,4 +1,4 @@
-name: Special group transactables
-custom_attributes:
+name: special_group_transactables
```

Comment:
@lemingos @Slashek Do we still want to have orphaned relationhips in docs?

Reply:
we can remove in next PR